src/visor/parameters.go

```diff
@@ -0,0 +1,130 @@
+package visor
+
+/*
+* CODE GENERATED AUTOMATICALLY WITH FIBER COIN CREATOR
+* AVOID EDITING THIS MANUALLY
+ */
+
+const (
```

Comment:
Should have some more sanity checks in an init() method in visor somewhere to make sure these values makes sense